src/index.html

```diff
@@ -0,0 +1,156 @@
+---
+title: Home
+---
+
+{% include 'head.html' %}
+
+<header>
+  <div>
+    <img src="/img/ipfs-standards.svg" alt="Interplanetary Standards Logo" width="60" height="60" class="logo">
+    <h1>IPFS Standards</h1>
+    <p>
+      The purpose of <em>IPFS Standards</em> is to foster interoperability between independent implementations of
+      the IPFS stack by producing Internet-grade specifications and test suites.
+    </p>
+  </div>
+</header>
+
+<main>
+  <h2 class="pitch-title">Specifying IPFS and the Interplanetary stack.</h2>
+  <p class="pitch-subtitle">
+    The technology that powers the content-addressable web is being standardized here.
+  </p>
+  <!--
+  <div class="basic-grid">
+    <div class="pitch">
+      <h3>High-quality open standards</h3>
+      <p>
+        We aim to fully define implementation behaviour and to empower anyone to build their
+        own system that fits into the interplanetary ecosystem without needing to reverse-engineer
+        the behavior of other implementations.
+      </p>
+    </div>
+    <div class="pitch">
+      <h3>Comprehensive test suites</h3>
+      <p>
+        A specification is only as useful as it is both testable and tested. We are building extensive
+        test suites that allow specifications to prove their interoperability and implementations to demonstrate
+        their correctness.
+        </p>
+    </div>
+    <div class="pitch">
+      <h3>Testing infrastructure</h3>
+      <p>
+        We build tools to support detecting interoperability issues early and often as part of CI/CD pipelines
+        and to report on implementation maturity as the stack evolves so that people who build on top of the
+        interplanetary stack can know what they can use in production.
+      </p>
+    </div>
+    <div class="pitch">
+      <h3>An open governance model</h3>
+      <p>
+        We guarantee community ownership of the standards, of the standards-production process, and of
+        the infrastructure to operate it, in collective pursuit of efficient, transformative, ethical
+        technology.
+      </p>
+    </div>
+  </div>
+  -->
+
+  <section>
+    <h2>Specifications</h2>
+    <p>
+      The specifications are broken up into multiple areas that cover the stack.
+    </p>
+    <div class="basic-grid">
+      <section>
+        <h3><a href="/http-gateways/">HTTP Gateways</a></h3>
+        <p>
+          IPFS Gateway acts as a bridge between traditional HTTP clients and IPFS. Through the gateway, users can download files,
+          directories and other IPLD data stored in IPFS as if they were stored in a traditional web server.
+        </p>
+        <dl>
+          <dt><a href="/http-gateways/path-gateway/">Path Gateways</a></dt>
```

Comment:
Wouldn't it be better to have this as an include shared with the index of `/http-gateways/`? Otherwise these are going to go out of sync.

Reply:
Good idea